dip_server/dip_platform/logic/game/resolution/transform_logic.py

```diff
@@ -169,6 +193,38 @@ def build_pydip_retreat_command(retreat_map, player, unit_map, command_data):
     raise ValueError
 
 
+def build_pydip_adjustment_command(ownership_map, player, unit_map, command_data):
+    """
+    ownership_map -- PyDip OwnershipMap representing current owned territories
+    player -- PyDip Player issuing the command
+    unit_map -- Territory Name to PyDip Unit object in that territory
+    command_data -- {
+        type -- Type of command being issued (dictates other parameters)
```

Comment:
should the comment match reality? (e.g. unitType, instead of type?)

Reply:
Ah, yeah should be updated.

`type` is still in there (that's the type of command), but `unitType` is necessary for the create command to know which kind of unit is being created. Thanks for catching that.